library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -1101,4 +1102,16 @@ private int translateStatus(int status) {
             }
         }
     }
+
+    /**
+     * Restart will activate the downloads workflow in case that it was not already active
+     * <p/>
+     * A possible scenario: A client denies a download for a particular business rule and that
+     * rule does not apply any more. Calling this method will reactivate the downloads workflow,
+     * check the client rules and proceed if necessary
+     */
+    public void restart() {
```

Comment:
when would a client call this? Can the downloadmanager be clever and call restart itself?